pkg/assetfetcher/asset.go

```diff
@@ -339,7 +339,8 @@ func (a assetHistoryEvents) lastScannedTimestamp() (time.Time, error) {
 			if err != nil {
 				return latestTime, err
 			}
-			if t.After(latestTime) {
+			// also need to make sure the date is not a 0 value
+			if t.After(latestTime) && !t.IsZero() {
```

Comment:
Task: This check is unnecessary. At this point, `latestTime` will either be: 1) the initial zero value, so `t.After(latestTime)` would be false if t is also a zero value, or 2) `latestTime` will be non-zero, and t.After will still return false is t is the zero value.

Reply:
yes! you are indeed correct, good point. Fixed!